app/src/ui/dispatcher/dispatcher.ts

```diff
@@ -2104,6 +2104,13 @@ export class Dispatcher {
     return this.appStore._showPullRequest(repository)
   }
 
+  /**
+   * Show the provided pull request on github.com
```

Comment:
Since it's not just GitHub.com

```suggestion
   * Open a browser and navigate to the provided pull request
```